ui/components/navbar.js

```diff
@@ -1,8 +1,15 @@
+import PropTypes from 'prop-types';
+
 import React from 'react';
 import SearchContainer from './search/search';
 
 
-function Navbar() {
+export default function Navbar({ showSearch }) {
+  let searchContainer;
+  if (showSearch) {
+    searchContainer = <SearchContainer />;
+  }
+
```

Comment:
Now that there's some logic in this function, should we add a test?

Reply:
Yep! Just added some.